lib/account/cubits/address_state.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flex_storefront/checkout/models/address.dart';
+import 'package:flex_storefront/shared/bloc_helper.dart';
+
+class AddressState extends BlocState {
```

Comment:
I think we should have 2 blocs/cubits:
- one for the Address list (this one already exists)
- one the the new address form